CONTRIBUTION.md

```diff
@@ -1,94 +1,136 @@
-- Make sure commit messages are in the proper format (see below).
-- Push changes in a topic branch to a personal fork of the repository.
-- To make sure that your topic branch is in sync with the remote master branch,
-follow a [rebase workflow](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).
-- Submit a pull request to openshift/windows-machine-config-operator (see PR workflow below).
-- The PR must receive one `/lgtm` and one `/approve` comments from the maintainers of the project.
+Before opening a pull request, be sure that your changes have been tested, documented, and checked for style. Once
+you think your PR is ready for review, be sure to check that you have
 
-Thanks for contributing!
+- [fetched from upstream](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#fetching-changes-from-a-remote-repository)
+- [Rebased your changes against your root branch](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+- [Run the tests locally](docs/HACKING.md)
+- linted your changes with ```make lint```
+- ensured error messages are a single line
+- updated relevant documentation if your PR changes user functionality  
 
-### Format of the pull request (PR)
+### Format your commits
 
-- The PR header for a feature should be prefixed with the Jira issue. Example: `WINC-123:`
-- The PR header for a bug should be prefixed with the Jira Bug identifier. Example: `OCPBUGS-123:`
-- Correctly prefixing the PR header will automatically associate the PR with the Jira issue or bug.
-  - In the case of bug fix PRs, this will also automatically transition the associated Jira bug.
-    - Opening the PR: `ASSIGNED` --> `POST`
-    - PR merges: `POST` --> `MODIFIED`
-    - `MODIFIED` to the `ON_QA` transition will have to be done manually by the PR author.
-    - PR closes without merging, the Jira bug transitions back to NEW status.
-- The individual commit messages should not be prefixed.
-
-### Format of the commit message
+When making your changes, feel free to use whatever commit system you like, but when it’s time to open the pull request,
+be sure when rebasing that your commits follow the WMCO’s formatting requirements.
 
 We follow a convention for commit messages that is designed to answer two questions: what changed and why. The
 subject line should feature the what and the body of the commit should describe the why.
 
-The format can be described more formally as follows:
+A good commit message should look like
 
 ```
 [subsystem] <what changed>
-<BLANK LINE>
+<BLANK LINE> 
 <why this change was made>
-<BLANK LINE>
-<footer>
+<BLANK LINE> 
+<Footer>
 ```
-Example for a sample feature:
+
+A real world example would look like
+
 ```
 [docs] Add the Guidlines
 
-Cupcake ipsum dolor sit. Amet tart cheesecake tiramisu chocolate cake topping.
-Icing ice cream sweet roll. Biscuit dragée toffee wypas.
+adds the contribution guidelines for maintaining consistent style across the repo 
```

Comment:
This line is over 80 chars 

Also, capitalize and add a period like the last example